api/s2n.h

```diff
@@ -253,6 +253,7 @@ extern int s2n_connection_client_cert_used(struct s2n_connection *conn);
 extern const char *s2n_connection_get_cipher(struct s2n_connection *conn);
 extern int s2n_connection_is_valid_for_cipher_preferences(struct s2n_connection *conn, const char *version);
 extern const char *s2n_connection_get_curve(struct s2n_connection *conn);
+extern const char *s2n_connection_get_kem(struct s2n_connection *conn);
```

Comment:
I get there's a convention here but these methods are named weirdly for returning a name instead of the actual object.

Reply:
Decided to name this one correctly and open another follow up PR to rename the others.